examples/src/Components/RecordMovieComponent.tsx

```diff
@@ -43,10 +48,15 @@ const RecordMovieComponent = ({
     };
 
     return (
-        <div>
+        <div className="ui-container">
+            <button onClick={() => setRecordingEnabled()}>
+                {isRecordingEnabled ? "Disable" : "Enable"} Recording
+            </button>
```

Comment:
Moved this into the component so that recording related things are all contained inside the `ui-container` class.